apps/protocol-api/src/prisma/schema.prisma

```diff
@@ -38,6 +38,21 @@ model User {
   guild_users          GuildUser[]
   twitter_user         TwitterUser?
   discord_users        DiscordUser[]
+  tips                 Tip[]
+  split_contracts      UserSplitContract[]
+}
+
+model UserSplitContract {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @default(now()) @updatedAt
+  user_id   Int
+  user      User     @relation(fields: [user_id], references: [id])
+  chain_id  Int?
+  chain     Chain?   @relation(fields: [chain_id], references: [id])
+  address   String
```

Comment:
This would be the address of the contract? Just double checking

Reply:
correct!